Salary App/Features/SalaryForm/Model/Calculators/TaxCalculator.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TaxCalculator.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+enum TaxBracket {
+    case PersonalAllowance
```

Comment:
Enum cases should never start with a capital letter they should all be lower case i.e.

```

enum TaxBracket {
 case personalAllowance
....
}

Also are you even using this?

```